src/tests/Microsoft.SymbolStore.UnitTests/PEFileKeyGenerationTests.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.SymbolStore.KeyGenerators;
+using TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.SymbolStore.Tests
+{
+    public class PEFileKeyGenerationTests
+    {
+        readonly ITracer _tracer;
+
+        public PEFileKeyGenerationTests(ITestOutputHelper output)
+        {
+            _tracer = new Tracer(output);
+        }
+
+        public class MockPEFile
+        {
+            public string Path { get; }
+            public string FileName { get; }
+            public string Guid { get; }
```

Comment:
Nit: Calling this Guid seems misleading, I don't think it is ever a Guid?

Reply:
Agree. This should be TimestampAndSize or something along those lines.

Reply:
I renamed the field to `Id`, as it is what eventually ends up in the `Id` arg of building the SymbolStoreKey at https://github.com/dotnet/diagnostics/blob/e5f6b52bb66e75d1b56945b2d1a5773289e544e4/src/Microsoft.SymbolStore/KeyGenerators/KeyGenerator.cs#L164

Moreover, the mock pefiles added in this PR have their bytes manipulated so that the `Id` value is MOCKCLR (0x4D 0x4F 0x43 0x4B 0x43 0x4c 0x52) , MOCKDAC (0x4D 0x4F 0x43 0x4B 0x44 0x41 0x43), etc. so for these mock files the value really is just an identifier than a legitimate timestamp/size